addons/wowl/static/src/components/dropdown/dropdown.js

```diff
@@ -10,10 +10,16 @@ export class Dropdown extends Component {
     this.hotkeyService = useService("hotkey");
     this.hotkeyTokens = [];
     this.state = useState({ open: this.props.startOpen, groupIsOpen: this.props.startOpen });
+    this.ui = useService("ui")
+    this.myActiveEl = this.ui.activeElement
     // Close on outside click listener
-    hooks.useExternalListener(window, "click", this.onWindowClicked);
-    // Listen to siblings states
-    useBus(Dropdown.bus, "state-changed", this.onSiblingDropdownStateChanged);
+    hooks.useExternalListener(this.myActiveEl, "click", this.onWindowClicked);
```

Comment:
does it mean that if you open a dropdown in a modal, and click on the backdrop, the dropdown stays open?